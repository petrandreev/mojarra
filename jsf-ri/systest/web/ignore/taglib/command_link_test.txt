      <form id="form01" method="post" action="/jsf-systest/faces/taglib/command_link_test.jsp;jsessionid=66560F4ABCA57E8DA0E279515F1FC72B">
