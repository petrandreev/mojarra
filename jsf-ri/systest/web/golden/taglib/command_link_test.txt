



<html>
<head>
    
    
    <title>command_link_test.jsp</title>
</head>
<body>
    
    
      <form id="form01" method="post" action="/jsf-systest/faces/taglib/command_link_test.jsp">

        <a id="form01:hyperlink01" href="#" onclick="document.forms[0]['form01:hyperlink01'].value='form01:hyperlink01'; document.forms[0].submit()">My Link</a>
        <a id="form01:hyperlink02" href="#" onclick="document.forms[0]['form01:hyperlink02'].value='form01:hyperlink02'; document.forms[0].submit()">This is a String property</a>
        <a id="form01:hyperlink03" href="#" onclick="document.forms[0]['form01:hyperlink03'].value='form01:hyperlink03'; document.forms[0].submit()">RES-BUNDLE LINK</a>
        <a id="form01:hyperlink04" href="#" onclick="document.forms[0]['form01:hyperlink04'].value='form01:hyperlink04'; document.forms[0].submit()"><img src="duke.gif" /></a>
        <a id="form01:hyperlink05" href="#" onclick="document.forms[0]['form01:hyperlink05'].value='form01:hyperlink05'; document.forms[0].submit()"><img id="form01:_id4" src="resbundle_image.gif" alt="" /></a>
        
            
        <a id="form01:hyperlink06" href="#" onclick="document.forms[0]['form01:hyperlink06'].value='form01:hyperlink06';document.forms[0]['param1'].value='value1'; document.forms[0].submit()">Paramter Link</a>
      <input type="hidden" name="form01" value="form01" /><input type="hidden" name="form01:hyperlink02" /><input type="hidden" name="form01:hyperlink06" /><input type="hidden" name="form01:hyperlink04" /><input type="hidden" name="form01:hyperlink01" /><input type="hidden" name="form01:hyperlink03" /><input type="hidden" name="param1" /><input type="hidden" name="form01:hyperlink05" /></form>
    
</body>
</html>

